src/chttpd_auth_cache.erl

```diff
@@ -95,7 +95,8 @@ get_from_cache(UserName) ->
 %% gen_server callbacks
 
 init([]) ->
-    {ok, #state{changes_pid = spawn_changes(0)}}.
+    self() ! {start_listener, 0},
```

Comment:
I forgot to mention `#5` which I saw in some projects.

`erlang:send_after(10, self(), init).`

I wouldn't use it.